.env.prod.example

```
@@ -19,9 +19,14 @@ NCP_OBJECT_STORAGE_SECRET_KEY=
 FE_IMAGE=
 FE_IMAGE_NAME=
 
-# ---------- Crawler -----------
-BE_API_URL=
-CRON_SCHEDULE=
+# ---------- Crawler ----------
+CRAWLER_IMAGE=
+BE_API_URL=http://backend:3000
+CRAWLER_CRON_SCHEDULE=*/5 * * * *
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CRAWLER_CRON_SCHEDULE key should go before the CRAWLER_IMAGE key

(UnorderedKey)